packages/node_modules/@ciscospark/plugin-calendar/test/integration/spec/calendar.js

```diff
@@ -76,9 +76,31 @@ function makeMockMeeting(params) {
   }, params);
 }
 
+function postToWhistler(spark, payload) {
+  return spark.request({
+    method: `POST`,
+    service: `whistler`,
+    resource: `calendarNotification`,
+    body: payload,
+    qs: {
+      isinteg: typeof process.env.WDM_SERVICE_URL !== `undefined` && process.env.WDM_SERVICE_URL.indexOf(`wdm-a.wbx2.com`) < 0
+    }
+  })
+    .then((res) => {
+      // Test response is using our parameters
+      const createdMeeting = res.body;
+      assert.isDefined(createdMeeting);
+      assert.equal(createdMeeting.meeting.meetingIdentifier.id, payload.meetingIdentifier.id);
+      assert.equal(createdMeeting.userUUID, payload.userUUID);
+      assert.equal(createdMeeting.meeting.startDate, payload.meeting.startDate);
+      assert.equal(createdMeeting.meeting.endDate, payload.meeting.endDate);
+      return res;
+    });
+}
+
 describe(`plugin-calendar`, () => {
-  describe(`#list()`, () => {
-    let createdMeeting, creator, mccoy, meetingParams, spark, spock;
+  describe(`list meeting`, () => {
```

Comment:
Why the name change here? describe blocks should typically adhere to the form:

```js
describe(`<plugin name>`, () => {
  describe(`<class name>`, () => {
    describe(`#<function name>()`, () => {
    });
  });
});
```

so, these tests should be under

```js
describe(`plugin-calendar`, () => {
  describe(`Calendar`, () => {
    describe(`#list()`, () => {
    });
  });
});
```